### ISSUE TITLE
SPCPlay triggers several antiviruses in VirusTotal

### ISSUE THREAD

**Issue by dajinkosa** [dgrfactory/spcplay#51]
Apologies for not using Japanese, but watashi do not speak nihongu

Ever since version 2.19.2 (mostly because i am lazy and did not test versions before that) the spcplay zip archive / individual exe components trigger at least 1 antivirus in virustotal, if they provide detail it's usually a trojan.

Example scans:
2.19.2 zip - https://www.virustotal.com/gui/file/a7885ac661c9105f322275175af9de9570c287e97ed63bfca8f4379f616bc996
2.19.3 zip - https://www.virustotal.com/gui/file/e92c0df5c94f53088f0d62cd0e6adeb2bcfe6a7c50088b43db6a889b2d44193d
2.19.4 zip - https://www.virustotal.com/gui/file/be8ddd2bbf14e0e64053bb212ba790292938124dee558de19c468197c9d43bee
2.19.4 spcplay.exe - https://www.virustotal.com/gui/file/92aa50575eeef2eb0cc143762c73ba613db970bfdcdd9e619881879d793f40a6

Steps to reproduce:
1. Get an spcplay zip, spcplay exe or spccmd exe
2. Upload it to https://www.virustotal.com
3. Enjoy the malware!

**Comment by dgrfactory** [comment]
Hi, @ADM228 
Thank you for reporting.

I know that VirusTotal detects something that looks like malware.
My player is built with delphi, and some anti-virus software seems to judge it as malware just because it is built with delphi.

I scan with well-known anti-virus software such as Microsoft Defender, ESET, and more, but these software do not detect.
In the past, I have sent files to Microsoft to see if they contained malware, and they determined that they were "Not Detected."
Therefore, it is considered a false-positive.

However, you are probably worried about what VirusTotal has detected.
In such cases, I recommend using other players.
Here are some other players what use my improved SNESAPU.

* Kuroneko SPC - https://kurohane.net/seisanbutu.html
* Music Player - https://debu0510.ifdef.jp/softr/souko.html
* KbMedia Player - https://kobarin.sakura.ne.jp/
* hoot - http://dmpsoft.s17.xrea.com/

**Comment by jeffythedragonslayer** [comment]
Yeah, this is an issue with VirusTotal, not SPCPlay.  @ADM228 can this be closed?

**Comment by ZaCloud** [comment]
Just updating the situation: on my Windows 10 desktop, Windows Defender now identified the spcplay-2.19.1.7540-ENG.zip version of the file as "Program:AndroidOS/Multiverze", & automatically quarantined it. My computer got really stuck & File Explorer crashed as soon as the file was downloaded & I had it queued to "open when completed" on Chrome browser. I couldn't even open Task Manager, my PC was so stuck & broken-looking for over 6 minutes. 

Not sure if it was just the scan itself using up all of the resources, or something else...

Strangely, my GPU was eating through half its RAM when normally on desktop operations it's pretty much dormant... & the Chrome tab I had open to download said file (https://www.smwcentral.net/?p=section&a=details&id=28962) was using over 1 GB of RAM just by itself. o_O (All this was only observable when my computer started moving again, & let me look at the graphs on Task Manager along with my RainMeter widgets. So it might have been even worse than that when it was stuck.)

Just letting y'all know about this... Could there be any danger that this file is affected by bad actors using one of the recently discovered vulnerabilities of libraries referencing one another or something? (I know almost nothing about how GitHub & stuff works, but just thought I'd take the time to bring it up just in case.)

I'm gonna let it get rid of the file just to be safe, in the meantime, & try something else. Thanks.

**Comment by jeffythedragonslayer** [comment]
> one of the recently discovered vulnerabilities of libraries referencing one another or something?

Do you have an example?

I don't recommend using that option "open when completed" - my PC gets busy reading something when I tell it to, not when the internet tells it to.

**Comment by dgrfactory** [comment]
Hi, @ZaCloud

To begin with... "spcplay-2.19.1.7540-ENG.zip" is **NOT** an official package.
It is on smwcentral.net? I did not upload to that site, it is **someone else repacked.**
Therefore, I am not involved in that zip file.

As @jeffythedragonslayer written, maybe tedious,
but more safe to use separate "download" and "open" operation in general terms.

**Comment by tcdw** [comment]
@dgrfactory I checked the version on SMW Central. Sure enough, the zip file is packed by someone else, and Windows Defender complains about `Program:AndroidOS/Multiverze`.

All of files presented in SMW Central's zip file are binary identicial to the [original release](https://github.com/dgrfactory/spcplay/releases/tag/2.19.1.7540), but with an English version of readme file added.

Now, I submitted that zip file to https://www.microsoft.com/en-us/wdsi/filesubmission for false positive.
